### PR TITLE
rework main and move to klog

### PR DESCRIPTION
**Pull request by luthermonson**
* rework main handle func to return errors
* migrate from glog to klog
* move app/linode dir to ./
* upgrade golangci-lint to v1.55.x

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,25 +4,24 @@ go 1.20
...
 	github.com/container-storage-interface/spec v1.3.0
-	github.com/golang/glog v1.1.2
-	github.com/golang/protobuf v1.5.2
 	github.com/linode/linodego v1.23.0
 	golang.org/x/net v0.17.0
 	golang.org/x/sys v0.13.0
 	google.golang.org/grpc v1.53.0
+	google.golang.org/protobuf v1.28.1
 	k8s.io/apimachinery v0.19.2
+	k8s.io/klog/v2 v2.110.1
```
Comment (okokes-akamai) [inline go.mod:13]: What's the reason behind the change?

**Review by shanduur — APPROVED** [review]
LGTM

File: go.mod (hunk 1 above)
Comment (okokes-akamai) [reply go.mod:13]: OK, fine.
